packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -809,8 +809,6 @@ class _IOSSimulatorDevicePortForwarder extends DevicePortForwarder {
 
   @override
   Future<void> dispose() async {
-    for (final ForwardedPort port in _ports) {
-      await unforward(port);
-    }
+    _ports.clear();
```

Comment:
I'd add a comment here that this only works because `unforward` is trivial. If `unforward` had to do some work, then `dispose` would have to call `unforward` on each port.

Reply:
iterated through a copy instead, PTAL.